Modules/Microsoft365DSC/Examples/Resources/IntuneDeviceConfigurationHealthMonitoringConfigurationPolicyWindows10/1-IntuneDeviceConfigurationHealthMonitoringConfigurationPolicyWindows10-Example.ps1

```diff
@@ -0,0 +1,34 @@
+<#
+This example is used to test new resources and showcase the usage of new resources being worked on.
+It is not meant to use as a production baseline.
+#>
+
+Configuration Example
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [PSCredential]
+        $Credscredential
+    )
+    Import-DscResource -ModuleName Microsoft365DSC
+
+    node localhost
+    {
+        IntuneDeviceConfigurationHealthMonitoringConfigurationPolicyWindows10 'Example'
+        {
+            AllowDeviceHealthMonitoring       = "enabled";
+            Assignments                       = @(
+                MSFT_DeviceManagementConfigurationPolicyAssignments{
+                    deviceAndAppManagementAssignmentFilterType = 'none'
+                    dataType = '#microsoft.graph.allLicensedUsersAssignmentTarget'
+                }
+            );
+            ConfigDeviceHealthMonitoringScope = "bootPerformance,windowsUpdates";
```

Comment:
Same comment as in the schema for this parameter. It makes more sense if we use an array instead of a comma separated string.